File: app/src/main/java/com/example/foodorderingapp/PayoutActivity.java
package com.example.foodorderingapp;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import com.example.foodorderingapp.databinding.ActivityPayoutBinding;

public class PayoutActivity extends AppCompatActivity {
    private ActivityPayoutBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Inflate the binding
        binding = ActivityPayoutBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        // Set onClickListener for the place order button
        binding.placemyorder.setOnClickListener(view -> {
            CongratsBottomSheet bottomSheetDialogue = new CongratsBottomSheet();
            bottomSheetDialogue.show(getSupportFragmentManager(), "");
        });

        // Set onClickListener for the back button
        binding.imageButtonPayout.setOnClickListener(view -> finish());
    }
}

File: app/src/main/java/com/example/foodorderingapp/PayoutActivity.kt
package com.example.foodorderingapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.foodorderingapp.databinding.ActivityPayoutBinding

class PayoutActivity : AppCompatActivity() {
    private lateinit var binding: ActivityPayoutBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityPayoutBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.placemyorder.setOnClickListener {
            val bottomSheetDialogue = CongratsBottomSheet()
            bottomSheetDialogue.show(supportFragmentManager, "")
        }

        binding.imageButtonPayout.setOnClickListener {
            finish()
        }
    }
}